SendEmail/README.md

```diff
@@ -9,15 +9,16 @@ products:
 
 # Email Sample
 
-This sample application shows how the Azure Communication Services Email SDK can be used to build an email experience. This sample sends an email to the required recipients of any domain using [Email Communication Services resource](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/email/create-email-communication-resource)
+This sample sends an email to the selected recipients of any domain using an [Email Communication Services resource](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/email/create-email-communication-resource). 
+Although the "recipients" field is technically required, it is not a requirement that you send an email to the recipients you select.
```

Comment:
@moirf My apologies, I think I wasn't clear - this line was just an explanation of why you should use the word "selected" instead of "required" in line 12. I didn't mean to suggest that this explanation be added to the document. Line 13 can be removed.